External/FEXCore/Source/Interface/Core/JIT/x86_64/ALUOps.cpp

```diff
@@ -671,6 +671,53 @@ DEF_OP(Extr) {
   }
 }
 
+DEF_OP(PExt) {
```

Comment:
If you want this to be an easy implementation, just let it use pext directly.
We already require AVX minspec on x86 host.